services/web-app/src/components/buttons/reviewBtn.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+interface ReviewButtonProps {
+  onClick: () => void;
+}
```

Comment:
Can you not maybe use the basic button here instead of creating a new one. Since they look  the same. Just that this button is centered, and you could do that with styling in a div outside the button?

Reply:
Yeah sure, I'll just need to merge the changes from main quick :)